modules/stanford_image_styles_preview/src/Form/PreviewForm.php

```diff
@@ -0,0 +1,187 @@
+<?php
+
+namespace Drupal\stanford_image_styles_preview\Form;
+
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\file\Entity\File;
+use Drupal\image\Entity\ImageStyle;
+
+/**
+ * Class PreviewForm.
+ */
+class PreviewForm extends FormBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'preview_form';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $form = [];
+
+    $form['image'] = [
+      '#type' => 'managed_file',
```

Comment:
So during validation, I can save the file but it becomes a managed file that way as well. I'm unable to find the path to the file even if it was a temporary path. 
If we keep it as a file_managed field, the uploaded file _should_ be cleaned up during garbage cleanup.
Another option would be to eliminate the upload and just have a few stock images included in the module and allow switching the preview.